src/cpu.cpp

```diff
@@ -1001,6 +1007,18 @@ int cpu_support_mips_msa()
 #endif
 }
 
+int cpu_support_loongarch_lsx()
+{
+#if defined __ANDROID__ || defined __linux__
+#if __loongarch64
+    return g_hwcaps & HWCAP_LOONGARCH_LSX;
+#else
+    return 0;
+#endif
+#else
+    return 0;
+#endif
+}
```

Comment:
add cpu_support_loongarch_lasx() for lasx

Reply:
fix